src/main/java/uk/gov/hmcts/reform/sendletter/services/ftp/FtpAvailabilityChecker.java

```diff
@@ -1,20 +1,13 @@
 package uk.gov.hmcts.reform.sendletter.services.ftp;
 
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.stereotype.Component;
-
 import java.time.LocalTime;
 
-@Component
 public class FtpAvailabilityChecker implements IFtpAvailabilityChecker {
 
     private final LocalTime downtimeStart;
     private final LocalTime downtimeEnd;
 
-    public FtpAvailabilityChecker(
-        @Value("${ftp.downtime.from}") String downtimeFromHour,
-        @Value("${ftp.downtime.to}") String downtimeToHour
-    ) {
```

Comment:
did this stop working?

Reply:
Integration tests beans up fake checker. Since spring boot 2.1 you cannot override by default and `@Component` already beaned up this prior configuration was run by. So solution is to either enable bean override or properly bean up objects - moved to configuration so integration tests continue to work

Reply:
thanks for clarification